azure-toolkit-libs/azure-toolkit-common-lib/src/main/java/com/microsoft/azure/toolkit/lib/common/task/AzureTaskContext.java

```diff
@@ -89,7 +91,11 @@ public static void run(final Runnable runnable, AzureTaskContext context) {
                 assert Objects.equals(task, popped) : String.format("popped op[%s] is not the exiting async task[%s]", popped, task);
             });
         } catch (final Throwable throwable) {
-            AzureExceptionHandler.onRxException(throwable);
+            final Throwable rootCause = ExceptionUtils.getRootCause(throwable);
+            if (!(rootCause instanceof InterruptedIOException) &&!(rootCause instanceof InterruptedException)) {
```

Comment:
Could we swallow `InterruptedException` in all cases? Or add an exception filter in messager context (May default to filter out interrupt exceptions